### ISSUE TITLE
Correctly display all profile network properties

### ISSUE THREAD

**Issue by HaroldoUFRN** [warewulf/warewulf#1042]
### Steps to reproduce

Let there be a profile by the name of "icex".

```
sudo wwctl profile set --netname MPI --type infiniband --netdev ib0 icex
sudo wwctl profile list -a icex | grep MPI
```

### Error message

There is no error message.
Network device is not displayed at all.
Network type is displayed as "--".
Both properties (device and type) are present in nodes.conf with correct values.

### Information on your system

4.3.0-1.git_235c23c

NAME="Rocky Linux"
VERSION="8.9 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.9"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.9 (Green Obsidian)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2029-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-8"
ROCKY_SUPPORT_PRODUCT_VERSION="8.9"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.9"

### General information

- [X] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have captured and reported relevant error messages and logs

**Comment by mslacken** [comment]
This will be fixed in the upcoming 4.5.x release
```
wwctl profile list -a icex | grep MPI
  icex     NetDevs[MPI].Type    --       infiniband  
  icex     NetDevs[MPI].Device  --       ib0         
```

**Comment by anderbubble** [comment]
@HaroldoUFRN I concur with @mslacken. If you'd like to check out the upcoming release, look at the nightly at https://github.com/warewulf/warewulf/releases/tag/nightly.

Note: while the release date says Dec 7, 2023, the actual artifacts are updated nightly.

You can monitor the status of the next release at https://github.com/warewulf/warewulf/milestone/4.
